src/theme/src/components/pane.js

```diff
@@ -0,0 +1,9 @@
+export default function getPaneStyles() {
+  return {
+    baseStyle: {
+      transitionDuration: '150ms',
+      transitionProperty: 'box-shadow, transform',
+      transitionTimingFunction: `cubic-bezier(0.0, 0.0, 0.2, 1)`
+    }
+  }
```

Comment:
`elevation` `hoverElevation` and `activeElevation` props can introduce `boxShadow` + `transform` css properties.

We could pull this into a modifier, but it's fairly redundant and could easily conflict with base styles or another modifier's styles:

```js
hoverElevations: {
  0: {
    _hover: {
      boxShadow: theme.tokens.elevations.0
    }
  },
  1: {
    translate: 'translateY(-2px)',
    _hover: {
      boxShadow: theme.tokens.elevations.1
    }
  },
  2: {
    translate: 'translateY(-2px)',
    _hover: {
      boxShadow: theme.tokens.elevations.2
    }
  },
  3: {
    translate: 'translateY(-2px)',
    _hover: {
      boxShadow: theme.tokens.elevations.3
    }
  },
}
```

Reply:
If we allow access to props in the style config fn we could do:
```js
baseStyles: {
  boxShadow: theme.elevations[props.elevation] || undefined,

  _hover: {
    ...(props.hoverElevation ? {
      boxShadow: theme.elevations[props.hoverElevation],
      transform: `translateY(-2px)`
    } : {}) // this is a bit ugly, and could be pulled out into a function for clarity.
}
```